dockers/docker-base/Dockerfile.j2

```diff
@@ -23,6 +23,10 @@ RUN rm -rf               \
     /var/cache/man/*     \
     /usr/share/locale/*
 
+{% if APT_CACHER_NG_PROXY.strip() -%}
+        RUN echo 'Acquire::http::Proxy "{{APT_CACHER_NG_PROXY}}";' >> /etc/apt/apt.conf.d/02proxy
```

Comment:
>/etc/apt/apt.conf.d/02proxy [](start = 70, length = 27)

Does it impact runtime proxy? If yes, production switches may have no access to build environment at all.